app/views/static/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Rails React</title>
+</head>
+<body>
+  <div id="root"></div>
+  <%= javascript_pack_tag 'application' %>
```

Comment:
- [ ] As per the project requirements, you should create a `root` view as an entry-point for your front-end application, to achieve the result, you should create a `root` controller and serve the `root.html.erb` file view, which does contain the `root` element for holding your `SPA` components. Because of the `wrong` configuration for the `React` application inside the `rails` using the `js_budling` gem, your back-end is not communicating with the `front-end`, so please try to fix things and make it work, fortunately you can click [here](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a) to follow the proper setup for the application.